app/renderer/components/navigation/navigator.js

```diff
@@ -102,13 +102,18 @@ class Navigator extends ImmutableComponent {
       .map((extension) => extensionState.getBrowserActionByTabId(this.props.appState, extension.get('id'), activeTabId))
       .filter((browserAction) => browserAction)
 
-    let buttons = extensionBrowserActions.map((browserAction, id) =>
-      <BrowserAction
-        browserAction={browserAction}
+    let buttons = extensionBrowserActions.map((browserAction, id) => {
+      let tabAction = browserAction.getIn(['tabs', activeTabId.toString()])
+      if (tabAction) {
+        tabAction = tabAction.set('title', browserAction.get('title'))
+        tabAction = tabAction.set('base_path', browserAction.get('base_path'))
```

Comment:
@bridiver The tab-specific action object lacks a title and base_path. I'm plucking them off of the more general browser action object. Is this the right way to resolve their absence, or should we be adding them when the object is passing through Muon?